doc/md/fundamentals/2-types/4-tuples.md

```diff
@@ -44,17 +44,17 @@ let second = ghost.1; // 25
 
 Tuples are useful for returning multiple values from a function without requiring a separate data structure.
 
-```motoko no-repl
+```motoko
 func getUserInfo() : (Text, Nat) {
     ("Ghost", 30);
-}
+};
 
 getUserInfo();
 ```
 
 ## Nesting tuples
 
-Tuples can be used to represent coordinate pairs, allowing for structured calculations such as finding the gradient of a line. The gradient (or slope) between two points `$(x_1, y_1)$` and `$(x_2, y_2)$` is calculated using the formula:
+Tuples can be used to represent coordinate pairs, allowing for structured calculations such as finding the gradient of a line. The gradient (or slope) between two points $(x_1, y_1)$ and $(x_2, y_2)$ is calculated using the formula:
```

Comment:
I would move ## Nesting tuples to the end, since it relies on pattern matching of tuples which is only introduce later.